build/ci/cloudbuild.push.yaml

```diff
@@ -73,6 +73,28 @@ steps:
       - |
         ./script/update_staging_version.sh $SHORT_SHA
 
+  # This is a trigger to push website at website/mixer code HEAD and with the
+  # latest base cache.
+  - name: 'gcr.io/datcom-ci/deploy-tool'
+    entrypoint: 'bash'
```

Comment:
for my understanding, is there a way to invoke a bash-script (which then can be separately tested) instead of embedding stuff inline?

Reply:
It can be put in a bash script. That would be helpful If shared in multiple place, like "./script/deploy_gke.sh" is used here

Reply:
Then I might suggest preferring shell-scripts over inline stuff... for testability.